### ISSUE TITLE
[Feature Request] Baseball Bat Crafting Recipe

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#31215]
(WEB REPORT BY: morena REMOTE: 206.221.180.138:7777)
# Revision
550525ddac31c9d324ef8a0011ac33f5cb17074b

I feel as if the baseball bat crafting recipe is too little for what the bat itself is.
It being made out of wooden planks, and only one of them, just seems a bit ridiculous.

My suggestion is that it's changed to something made from our raw wood, perhaps using the chisel on the wood will craft a baseball bat. Would make it slightly harder to get.

**Comment by ManopolisJones** [comment]
I agree with the fact the bat is too cheap to make with how powerful it is, but I don't think we need to overthink things here. I'd say make it five planks of wood instead of one like the paddle. That way it takes at least a little more doing to get rather then removing a single plank from one of the boarded up doors to get your delinquent slugger.
